server/bundles/io.cloudbeaver.server/schema/service.core.graphqls

```diff
@@ -494,10 +501,14 @@ input ConnectionConfig {
 
     # Custom connection parameters (all optional)
 
-    host: String
-    port: String
-    serverName: String
-    databaseName: String
+    host: String @deprecated(reason: "24.1.1, use mainProperties")
+    port: String @deprecated(reason: "24.1.1, use mainProperties")
+    serverName: String @deprecated(reason: "24.1.1, use mainProperties")
+    databaseName: String @deprecated(reason: "24.1.1, use mainProperties")
```

Comment:
in `DriverInfo`:
```
    defaultHost: String
    defaultPort: String
    defaultDatabase: String
    defaultServer: String
  ```

must be marked as deprecated also, because now the default value can get from `mainProperties`

Reply:
we don't have a way to get this fields right, so keep it as it is